### PR TITLE
Feature/clp test tables

### PR DESCRIPTION
**Pull request by AustinoBombino**
fixed precision and rounding errors, removed duplicate entries, added double swap fees and slips

### CODE REVIEW

**Review by utx0 — APPROVED** [review]

File: go.sum [hunk 1]
```diff
@@ -108,6 +108,7 @@ github.com/cosmos/cosmos-sdk v0.39.1 h1:vhjf9PZh9ph8btAj9aBpHoVITgVVjNBpM3x5Gl/V
 github.com/cosmos/cosmos-sdk v0.39.1/go.mod h1:ry2ROl5n+f2/QXpKJo3rdWNJwll00z7KhIVcxNcl16M=
 github.com/cosmos/cosmos-sdk v0.39.2 h1:nLfCJMkUuFt7ansi/YvCxwwxLFrgHCA3cYP4sJKYQdk=
 github.com/cosmos/cosmos-sdk v0.39.2/go.mod h1:VNUluciWBFj2vkhpMcp8rYZL/kCw0FtNc7SseUjE1KM=
+github.com/cosmos/cosmos-sdk v0.40.1 h1:gjrtV3MQj/CMeyXN4+sosHMG6Xwa2uH6HITSjSNL/0E=
```
Comment (utx0) [inline go.sum:111]: Hey @AustinoBombino this needs to be rolled back to the correct version as this will break everything.

**Review by utx0 — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (AustinoBombino) [reply go.sum]: removed in 08ace4c

File: test/test-tables/pool_units.json [hunk 2]
```diff
@@ -1,4 +1,12 @@
 {
+  "_info": {
```
Comment (ryardley) [inline test/test-tables/pool_units.json:2]: Ha nice idea. I hate how you cannot have comments in json.

**Comment by ryardley** [comment]
Austin do you need anything done from a frontend perspective in this PR here?

File: test/test-tables/pool_units.json (hunk 2 above)
Comment (AustinoBombino) [reply test/test-tables/pool_units.json:2]: haha credit goes to @Jedi2002

**Comment by AustinoBombino** [comment]
> Austin do you need anything done from a frontend perspective in this PR here?

@ryardley no, not unless you guys want to make any alterations to the tables. If you're happy with them then we're good.

**Comment by ryardley** [comment]
>  no, not unless you guys want to make any alterations to the tables. If you're happy with them then we're good.

It just seems the FE tests are failing because one of the tables we were pointing to has been removed.

**Comment by AustinoBombino** [comment]
> > no, not unless you guys want to make any alterations to the tables. If you're happy with them then we're good.
> 
> It just seems the FE tests are failing because one of the tables we were pointing to has been removed.

all of the old tables have new analogues in this PR. the old tables had bad precision and rounding and scientific notation in places.

I could revert the names but then the names wouldn't be super clear as there are now tables dedicated to single and double swaps

**Comment by AustinoBombino** [comment]
the tradeslip table was removed because we're getting rid of tradeslip

**Review by Jedi2002 — DISMISSED** [review]

File: go.sum (hunk 1 above)
Comment (AustinoBombino) [reply go.sum]: @utx0 does this look good to you now?

**Comment by ryardley** [comment]
So I just wanted to check up on this did you need us to hook up the tables to frontend? Just that the tests are failing so we can't merge.

**Comment by AustinoBombino** [comment]
> So I just wanted to check up on this did you need us to hook up the tables to frontend? Just that the tests are failing so we can't merge.

Yeah if you could hook the tables up so that the tests pass that would be great.

**Review by utx0 — APPROVED** [review]

**Pull request merged**